_data/kaupungit.yml

```
@@ -11,11 +11,17 @@
   lat: 60.22199
   lon: 24.85194
   osoite: Takkatie 18, takaovi
+  officialname: Helsinki Hacklab ry
+  ytunnus: 2346395-6
+  regno: 203.110
```

```suggestion
  regno: 203.110
```

I'm not sure register numbers are that relevnat info anymore as everything is officially Y-tunnus now.

---

yeah, we can probably leave them out. I have never needed them for anything ever :)

---

and done in 9f9f3dd